util/envDynawo.sh

```diff
@@ -1593,6 +1593,7 @@ deploy_dynawo() {
   echo "deploying libiidm libraries"
   cp -P $DYNAWO_LIBIIDM_HOME/lib/*.* lib/
   if [ -d "$DYNAWO_LIBIIDM_HOME/bin" ]; then
+    mkdir -p bin
```

Comment:
I would put this right below mkdir -p lib (that is some lines above) and delete the mkdir -p bin located some lines below

Reply:
I will move it. I first decided to put it here so we only create the `bin` folder if it exists in IIDM, but I agree: it will be better to have it close to the creation of the `lib` folder